syntax = "proto3";
package role;





/*
GetUserRole请求
*/
message GetUserRoleRequest {
    /*
    用户名
    */
    string user = 1 ;
}

/*
GetUserRole返回
*/
message GetUserRoleResponse {
    /*
    角色名称列表
    */
    repeated string roles = 1 ;
}

/*
GetUserRoleApi返回
*/
message GetUserRoleResponseWrapper {
    /*
    返回码
    */
    int32 code = 1 ;
    /*
    返回码解释
    */
    string codeExplain = 2 ;
    /*
    错误详情
    */
    string error = 3 ;
    /*
    返回数据
    */
    role.GetUserRoleResponse data = 4 ;
}

